app/src/main/java/com/example/egida/presentation/ui/WorkFragment.kt

```diff
@@ -33,47 +32,23 @@ class WorkFragment : Fragment() {
     override fun onResume() {
         super.onResume()
         binding.minusValueWork.setOnClickListener {
-            lifecycleScope.launch {
-                viewModel.day
-                    .collect {
-                        it.work--
-                        binding.textWork.text = it.work.toString()
-                    }
-            }
+            dayViewModel.minusValueWork(binding.textWork)
```

Comment:
Never pass a view or context to ViewModel. Write correct data flow diagram with correct answers and you'll see a solution.